# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/coverage/simple.js
# Commit: eeaeb40f8b86
# Full Hash: eeaeb40f8b869121eff65b1c2d11af423c9a9ecc
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2022-03-09 09:44:44
# Regressor Bug: 885514
# File Overlap Count: 1
# Description:
#   Bug 885514: Simplify JSOp::Retsub r=jandem
#   
#   Currently, JSOp::Retsub takes two arguments, and uses one to decide whether to throw the second or use it as a resume index. This patch splits out the exception-handling logic into explicit bytecode, leaving JSOp::Retsub as a simple indirect jump.
#   
#   To make the generated bytecode work out nicely, I've swapped the order of the arguments: now `throwing` is on top, and the exception/resumeindex argument is underneath it.
# ==============================================================================

diff -r 1cb0b352b024 -r eeaeb40f8b86 js/src/jit-test/tests/coverage/simple.js
--- a/js/src/jit-test/tests/coverage/simple.js	Tue Mar 08 21:10:22 2022 +0000
+++ b/js/src/jit-test/tests/coverage/simple.js	Tue Mar 08 21:10:22 2022 +0000
@@ -242,8 +242,8 @@
   //FNH:1
   //LF:10
   //LH:9
-  //BRF:2
-  //BRH:1
+  //BRF:4
+  //BRH:2
 });
 
 checkLcov(function () { //FN:$,top-level //FNDA:1,%